toolkit/smart-contracts/plutus-data/Cargo.toml

```diff
@@ -17,9 +17,9 @@ sidechain-domain = { workspace = true, features = ["std"] }
 sp-core = { workspace = true }
 cardano-serialization-lib = { workspace = true }
 thiserror = { workspace = true }
+raw-scripts = { workspace = true }
```

Comment:
`raw-scripts` are implicit dependency of production code. But I admit I moved it here to get rid of warning in docs of production code only.